lib/LocalBinary.js

```diff
@@ -78,7 +78,7 @@ function LocalBinary(){
   this.binaryPath = function(conf, callback){
     var destParentDir = this.getAvailableDirs();
     var destBinaryName = (this.windows) ? 'BrowserStackLocal.exe' : 'BrowserStackLocal';
-    var binaryPath = path.join(destParentDir, destBinaryName);
+    var binaryPath = '"' + path.join(destParentDir, destBinaryName) + '"';
```

Comment:
Isn't the sanitization of the path can be handled by this [function](https://github.com/browserstack/browserstack-local-nodejs/blob/f4ce0bb7385792632874db65f1365894be38c55d/lib/Local.js#L11)